### PR TITLE
fix: remove duplicate items on populate query

### PR DESCRIPTION
**Pull request by cdoak-gh**

### CODE REVIEW

File: workflows/ConfiguredQuery/content.json [hunk 1]
```diff
@@ -3626,11 +3659,13 @@
             "id": 730
         }
     ],
-    "licenseInfo": { "licenseeId": "88cb61d6-af47-9b3e-5181-ed674c4c0815" },
+    "licenseInfo": {
+        "licenseUrl": "https://vn-demo.vertigis.com/vertigisstudio/workflow/service/auth/license"
```
Comment (rcooney) [inline workflows/ConfiguredQuery/content.json]: We should remove the `licenseInfo`

File: src/activities/AddQueryElements.ts [hunk 2]
```diff
@@ -344,8 +344,17 @@ export default class AddQueryElements implements IActivityHandler {
                 );
                 items = collItems.items;
             } else {
+                const filterDistinctValue = Array.from(
+                    new Set(results.features.map(
+                        (item: __esri.Graphic) => {
+                            const clone = item.clone();
+                            clone.attributes = { [searchField.field]: item.attributes[searchField.field] };
+                            return clone;
```
Comment (rcooney) [inline src/activities/AddQueryElements.ts]: How does this make it distinct? Doesn't the `clone()` ensure that different instances are passed to the `Set`, so it won't actually collapse any duplicates??

**Review by rcooney — CHANGES_REQUESTED** [review]

File: workflows/ConfiguredQuery/content.json (hunk 1 above)
Comment (rcooney) [reply workflows/ConfiguredQuery/content.json]: If it is in the other workflow please remove it there too.

Comment (cdoak-gh) [reply workflows/ConfiguredQuery/content.json]: Fixed

File: src/activities/AddQueryElements.ts (hunk 2 above)
Comment (cdoak-gh) [reply src/activities/AddQueryElements.ts]: You're correct.  It didn't work at all.  I had some additional logic in the Workflow that also filters that masked the fact.  It's fixed now.

**Review by rcooney — APPROVED** [review]

**Pull request merged**